### ISSUE TITLE
Background colors not showing in Windows

### ISSUE THREAD

**Issue by cloudrex** [cloudrex/discord-term#4]
Seems like the background colors aren't showing on Windows CMD.

I guess I'll have to try to do some fixing here-and-there and get it to work.

Even if it doesn't work, I can make it use borders instead of backgrounds in Windows only, tho I'm pretty sure with some fixing it should work.

**Comment by cloudrex** [comment]
My friend @sirhypernova has confirmed background colors to work in Windows CMD (Windows 10).

Although it didn't work for me, I'd have to retry it.

**Comment by duttaditya18** [comment]
Works for me in PowerShell but not in CMD.

**Comment by ell10t** [comment]
Seems to only draw white and black backgrounds for me, running 17134.

**Comment by ell10t** [comment]
I take it back, ruby and such works fine with CMD. It just seems to be very limited for color options, for example:

<img width="300" src="https://user-images.githubusercontent.com/35668113/54130268-fbb65080-4407-11e9-9aee-2e4ace266b2a.png">

**Comment by cloudrex** [comment]
@ell10t Yes, Windows CMD's color scheme is horrible. I'll see what I can do about it.
